### DISCUSSION TITLE
请问一下关于flex+bison与现在用的meta parser的优劣？

### DISCUSSION THREAD

**Discussion by rocketman123456** [BoomingTech/Piccolo#287] [General]
请问一下关于flex+bison与现在用的meta parser的优劣？

**Comment by zDagwood** [comment]
flex+bison可以自己定制实现一个语法分析器，然后生成代码。
在meta parser里，语法分析器这部分是直接交给clang的，不需要自己重新实现一遍C++的整套语法解析。
生成代码这部分，两者都要做，只不过输入和生成工具不同，本质区别不大
